src/lib/panorama.ts

```diff
@@ -259,147 +267,187 @@ function getTileUrl(pano: Panorama, x: number, y: number, zoom: number) {
 	}
 }
 
-type PanoMetadata = NonNullable<Awaited<ReturnType<typeof getPanoMetadataFromId>>>;
-export function renderPanoFromMetadata(
+const envCanvas = document.createElement('canvas');
+const envContext = envCanvas.getContext('2d')!;
```

Comment:
Get the one for two combo
```suggestion
const envContext = document.createElement('canvas').getContext('2d')!;
```

then you can access the canvas with `envContext.canvas`